frontend/src/components/tickets-form/product-row.tsx

```diff
@@ -170,6 +175,15 @@ export const ProductRow = ({
           </div>
         )}
 
+        {!ticket.soldOut && isMembershipProduct && (
```

Comment:
This `ticket` is the one for the association subscription? Can be it SoldOut? 
If the tickets are sold out, people can't subscribe from this page to the association?

Reply:
Nope, it can't go sold-out, the soldOut check is there only because I copied and pasted the code.

The whole tickets page is currently a huge mess so I want to refactor it in the future